trade-finance-manager-ui/server/routes/case/index.js

```diff
@@ -37,6 +37,9 @@ router.get('/:_id/parties/bond-beneficiary', partiesController.getBondBeneficiar
 router.post('/:_id/parties/bond-beneficiary', caseController.postTfmFacility);
 
 router.get('/:_id/activity', activityController.getActivity);
+router.post('/:_id/activity', activityController.filterActivities);
+router.get('/:_id/activity/post-comment', activityController.getCommentBox);
+router.post('/:_id/activity/post-comment', activityController.postComment);
```

Comment:
unit test in the same directory needs the new routes added

Reply:
done